src/csim/stat_ops_expectation_value.cpp

```diff
@@ -171,13 +555,36 @@ double expectation_value_multi_qubit_Pauli_operator_partial_list(
         Pauli_operator_type_list, target_qubit_index_count, &bit_flip_mask,
         &phase_flip_mask, &global_phase_90rot_count, &pivot_qubit_index);
     double result;
+
+#ifdef _USE_SVE
+    // # of complex128 numbers in an SVE register
+    ITYPE VL = svcntd() / 2;
+#endif
+
     if (bit_flip_mask == 0) {
-        result = expectation_value_multi_qubit_Pauli_operator_Z_mask(
-            phase_flip_mask, state, dim);
+#ifdef _USE_SVE
+        if (dim >= VL) {
```

Comment:
```suggestion
        if (dim > VL) {
```
`expectation_value_multi_qubit_Pauli_operator_Z_mask_sve()` processes `VL<<1` elements in one step, doesn't it?

Reply:
Thanks for the code review. The condition seems incorrect, and we will check them, including tests.

Reply:
We have corrected the three conditions you pointed out.